### PR TITLE
Cleanup adapters

### PR DESCRIPTION
**Pull request by supersonicclay**
This is to cleanup all the adapter functions that were in place to ease the refactor from the more stateful ModeHandlers to the more stateless EditModes interface.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -18,12 +18,13 @@
...
-    "@luma.gl/constants": "^8.0.1",
-    "@luma.gl/core": "^8.0.1",
-    "@luma.gl/webgl": "^8.0.1",
+    "@deck.gl/core": "^7.0.2",
+    "@deck.gl/layers": "^7.0.2",
+    "@deck.gl/react": "^7.0.2",
+    "@luma.gl/addons": "^7.0.0",
+    "@luma.gl/constants": "^7.0.0",
+    "@luma.gl/core": "^7.0.0",
+    "@luma.gl/webgl": "^7.0.0",
```
Comment (georgios-uber) [inline website/package.json]: 😨

Comment (supersonicclay) [reply website/package.json]: I just did this temporarily until we get it working with deck.gl 8. I'll fix the deck.gl 8 integration first before landing this.

**Comment by supersonicclay** [comment]
@georgios-uber this is ready for review

**Comment by supersonicclay** [comment]
@georgios-uber sorry for the large diff. It's hard to do these refactors piecemeal. I can pull out a couple parts into separate PRs if you need.

**Pull request merged**